source/common/http/http1/parser_impl.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <memory>
+
+#include "common/http/http1/parser.h"
+
+namespace Envoy {
+namespace Http {
+namespace Http1 {
+
+class HttpParserImpl : public Parser {
```

Comment:
Small tests for HttpParserImpl?

Reply:
minimal tests added with a no-op parser callback.